app/services/stats/management_information/queries/base_count_query.rb

```diff
@@ -12,10 +12,11 @@ def self.call(**kwargs)
         new(kwargs).call
       end
 
-      def initialize(date_range:, date_column_filter:)
+      def initialize(date_range:, date_column_filter:, replace_journeys_func: false)
```

Comment:
lets remove the instance level logic and just use the app flag as a trigger

Reply:
done